observatory-platform/observatory/platform/cli/templates/generate_project/setup.cfg.jinja2

```diff
@@ -0,0 +1,50 @@
+{# Copyright 2020 Curtin University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Author: Aniek Roelofs -#}
+[metadata]
+name = {{ package_name }}
+python-requires = >=3.7
```

Comment:
This version should be templated out.  You could hard code the version in the Python driver code that generates the template for the time being.  The installation process work is moving towards centralising these types of version requirements.